src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <form @submit.prevent="editCategory">
+    <app-heading :level="1">カテゴリー管理</app-heading>
+    <app-router-link
+      class="category-management-edit__link"
+      underline
+      small
+      hover-opacity
+      :to="`/categories`"
+    >
+      カテゴリー一覧へ戻る
+    </app-router-link>
+    <app-input
+      v-validate="'required'"
+      name="category"
+      type="text"
+      placeholder="更新するカテゴリー名を入力してください"
+      data-vv-as="カテゴリー名"
+      :error-messages="errors.collect('category')"
+      :value="category"
+      @update-value="$emit('update-value', $event)"
+    />
+    <app-button
+      class="category-management-edit__submit"
+      button-type="submit"
+      round
+      :disabled="disabled || !access.create"
+    >
+      {{ buttonText }}
+    </app-button>
+
+    <div v-if="errorMessage" class="category-management-edit__notice">
+      <app-text bg-error>{{ errorMessage }}</app-text>
+    </div>
+
+    <div v-if="doneMessage" class="category-management-edit__notice">
+      <app-text bg-success>{{ doneMessage }}</app-text>
+    </div>
+  </form>
+</template>
+<script>
```

Comment:
[nits]

template, script, style 間は1行空けた方が見やすくなるので修正をお願いします。

```suggestion
</template>

<script>
```

Reply:
@Sashimi-tt 
scriptタグとstyleタグの間も1行開けましょう！